settings.gradle

```diff
@@ -1,3 +1,4 @@
 include 'safe-logging'
 include 'preconditions'
+include 'preconditions-assertj'
```

Comment:
Might be worth a cheeky little README entry to include a snippet of usage?  Also telling people they can depend on `com.palantir.logsafe:preconditions-assertj`?